front/components/layout/MobileHeader.vue

```diff
@@ -5,7 +5,7 @@
     <v-container fluid style="padding: 0">
       <v-row style="padding: 0 8px">
         <template v-if="!searchIdeaMode">
-          <v-col style="padding: 0" cols="10">
+          <v-col style="padding: 0; padding-top: 0.5rem;">
```

Comment:
@f-prime i checked out all your commits here. they look good.

note, just since it relates to this particular changed line:

vuetify is the css framework i'm using for this.
it has helper classes like px-1 (padding x 1rem) and mb-4 (margin bottom 4rem)
i don't expect you to know that, so this isn't a knock, just letting you know for future reference.

i've been slowly stripping out all the inline CSS from the previous devs.

as you start working with this codebase, when you see bad decisions and practices, please refactor if you feel comfortable (or ticket for later refactoring)